deploy/kubernetes/csi-snapshotter/setup-csi-snapshotter.yaml

```diff
@@ -72,7 +72,7 @@ spec:
       serviceAccount: csi-snapshotter
       containers:
         - name: csi-provisioner
-          image: quay.io/k8scsi/csi-provisioner:v1.5.0-rc1
+          image: quay.io/k8scsi/csi-provisioner:v1.5.0
```

Comment:
In the other repos, we use "latest" tag so we don't need to remember to update it every release

Reply:
Sure.  I want to backport this to release-2.0 branch.  Do you suggest to submit a PR directly to the release-2.0 branch?

Reply:
I guess this is a strange example because in other sidecar repos, the deployment is not a complete/workable spec and the expectation is to go to the hostpath driver repo to look at a real example with fixed versions

Reply:
We want to provide an example here so people can test create snapshot easily without deploying a full blown setup as in the hostpath driver repo.  I can add a note here for people to look at that repo for full testing of all features.